### ISSUE TITLE
FDM Printing?

### ISSUE THREAD

**Issue by mattkerrison** [maxbbraun/pisight#12]
I've bought a used iSight & have the Raspberry pi lying around, but I was curious if anyone has tried to print the enclosure with an FDM printer? 
With a 0.2mm nozzle I'd think it would likely be possible, curious as I would prefer that than ordering a resin printed piece.

**Comment by phenomeus** [comment]
same for me and also I bought an ender 3 v2 and will try to print this. right now I play around with Cura slicer. will report asap, but right now I have too much projects going on in parallel

**Comment by stockhuman** [comment]
_Just_ finished completing this project, printed this on an Ender 3.

I had to omit the tiny fasteners due to the very tight tolerances needed around the shutter assembly & the two anchoring points to the cylindrical case. It's effectively friction-fit, but that works rather well. A dremel comes in handy.
![100062500152_23081](https://user-images.githubusercontent.com/13852458/105792463-67175600-5f7f-11eb-8b0f-4b280dd453bc.jpg)

**Comment by phenomeus** [comment]
what settings on the ender? standing upright?

what adhesion setup? skirt? brim? raft? 

infill? suport?

so many questions :D I will do it on my ender 3 v2 asap, but I still have so many projects going on

**Comment by stockhuman** [comment]
@phenomeus 

That's right, standing tall, printed with the camera-insert side touching the build plate. I did this to minimize supports, which were a pain to remove.

Simple skirt around the print, no special adhesion settings. And I believe 14% infill (gyro), nearly default (Cura HQ) settings otherwise. No part cooling fan, but that's specific to my machine.

Good luck on your print. The real pain was assembly!

**Comment by phenomeus** [comment]
finally I got it printed on my ender 3 v2 with 10% support that I had to pull off. see pictures, everything went fine
but I had the camera side pointing to the top. 

![IMG_8569](https://user-images.githubusercontent.com/5281493/113202582-35668800-926b-11eb-93a6-f276f7f3ac1e.jpeg)
![IMG_8570](https://user-images.githubusercontent.com/5281493/113202589-38617880-926b-11eb-8638-267bf020bd54.jpeg)
![IMG_8571](https://user-images.githubusercontent.com/5281493/113202593-3992a580-926b-11eb-8dd3-1981db9b436e.jpeg)
![IMG_8572](https://user-images.githubusercontent.com/5281493/113202597-3ac3d280-926b-11eb-9981-c6f19cad51ca.jpeg)
![IMG_8573](https://user-images.githubusercontent.com/5281493/113202602-3b5c6900-926b-11eb-8090-2f5d6149181a.jpeg)
![IMG_8574](https://user-images.githubusercontent.com/5281493/113202606-3bf4ff80-926b-11eb-9ea3-43712d351e6f.jpeg)

**Comment by msgilligan** [comment]
> with 10% support that I had to pull off. see pictures

Do you have any photos with the support still attached? I'd be curious to see what it looked like at that point.

My Ender 3 V2 is supposed to arrive today and I am new to 3D printing, but this is one of the first projects I'm going to try.

**Comment by phenomeus** [comment]
sorry I don't but here you can see the green "helpers" = support that I had to remove 

<img width="908" alt="Bildschirmfoto 2021-03-31 um 22 10 20" src="https://user-images.githubusercontent.com/5281493/113204965-00a80000-926e-11eb-96db-d9f0ca1ff175.png">
<img width="796" alt="Bildschirmfoto 2021-03-31 um 22 10 28" src="https://user-images.githubusercontent.com/5281493/113204981-04d41d80-926e-11eb-8148-fe6baf1fcf68.png">

**Comment by msgilligan** [comment]
@phenomeus Thanks!

**Comment by phenomeus** [comment]
your welcome, here you can see my finished build https://github.com/maxbbraun/pisight/issues/16
